src/store/hooks.js

```diff
@@ -0,0 +1,13 @@
+import { useContext } from "react";
+
+import { GlobalContext } from "./GlobalState";
+
+export function useGlobalFunctions() {
+  const { globalFunctions } = useContext(GlobalContext);
+  return globalFunctions;
+}
+
+export function useGlobalState() {
+  const { globalState } = useContext(GlobalContext)
+  return globalState;
+}
```

Comment:
these functions decreased readability for me later in the code. I had no idea what they were doing, so i had to look them up. do they serve a specific purpose?

Reply:
They were intended to improve readability :stuck_out_tongue_closed_eyes:

These are custom hooks, so they follow the `useSomeCustomHook` naming pattern. Basically a custom function that can call other hooks. These two here are not strictly necessary, so we can also easily get rid of them.